packages/addons/tools/system-tools/package.mk

```diff
@@ -70,6 +71,10 @@ addon() {
     # diffutils
     cp -P $(get_install_dir diffutils)/usr/bin/{cmp,diff,diff3,sdiff} ${ADDON_BUILD}/${PKG_ADDON_ID}/bin
 
+    # dislocker
+    cp -P $(get_install_dir dislocker)/usr/bin/dislocker{,-bek,-file,-fuse,-metadata} ${ADDON_BUILD}/${PKG_ADDON_ID}/bin
+    cp -P $(get_install_dir dislocker)/usr/lib/libdislocker.so{,.0.7{,.2}} ${ADDON_BUILD}/${PKG_ADDON_ID}/lib
```

Comment:
- please make this more maintainable (e.g. *)
- add required `libmbedtls` depoending on the final build of libraries